analyzer/os/redhatbase/redhatbase_test.go

```diff
@@ -0,0 +1,63 @@
+package redhatbase
+
+import (
+	"reflect"
+	"testing"
+
+	"golang.org/x/xerrors"
+
+	"github.com/knqyf263/fanal/analyzer"
+	"github.com/knqyf263/fanal/analyzer/os"
+)
+
+func TestAnalyze(t *testing.T) {
+	var tests = map[string]struct {
+		path    string
+		os      analyzer.OS
+		wantErr error
+	}{
+		"CentOS": {
+			path: "./testdata/centos",
+			os:   analyzer.OS{Family: os.CentOS, Name: "7.6.1810"},
+		},
+		"Fedora29": {
+			path: "./testdata/fedora_29",
+			os:   analyzer.OS{Family: os.Fedora, Name: "29"},
+		},
+		"Fedora31": {
+			path: "./testdata/fedora_31",
+			os:   analyzer.OS{Family: os.Fedora, Name: "31"},
+		},
+		"Oracle7": {
+			path: "./testdata/oracle_7",
+			os:   analyzer.OS{Family: os.Oracle, Name: "7.6"},
+		},
+		"Redhat6": {
+			path: "./testdata/redhat_6",
+			os:   analyzer.OS{Family: os.RedHat, Name: "6.2"},
+		},
+		"Invalid": {
+			path:    "./testdata/not_redhatbase",
+			wantErr: os.AnalyzeOSError,
+		},
+	}
+	a := redhatOSAnalyzer{}
+	for i, v := range tests {
```

Comment:
`i` seems to be index. `name` or `testname` is better.

Reply:
fixed